docs/reference/ingest/processors/geoip.asciidoc

```diff
@@ -4,21 +4,20 @@
 <titleabbrev>GeoIP</titleabbrev>
 ++++
 
-The `geoip` processor adds information about the geographical location of IP addresses, based on data from the Maxmind databases.
-This processor adds this information by default under the `geoip` field. The `geoip` processor can resolve both IPv4 and
```

Comment:
I removed the sentence about the `geoip` field. This is covered in the parameter docs below.